File: tandem_bicycle.py
def tandemBicycle(redShirtSpeeds, blueShirtSpeeds, fastest):
    redShirtSpeeds.sort()
    blueShirtSpeeds.sort(reverse=fastest)

    return sum(
        [max(redShirtSpeeds[i], blueShirtSpeeds[i]) for i in range(len(redShirtSpeeds))]
    )

File: tandem_bicycle.js
function tandemBicycle(redShirtSpeeds, blueShirtSpeeds, fastest) {
  redShirtSpeeds.sort((a,b) => b - a);
  blueShirtSpeeds.sort((a,b) => fastest ? a - b : b - a);

  let result = 0;

  for (let index = 0; index < redShirtSpeeds.length; index++) {
    result += Math.max(redShirtSpeeds[index], blueShirtSpeeds[index]);
  }
  return result;
}

exports.tandemBicycle = tandemBicycle;